### ISSUE TITLE
Incompatible with latest casacore

### ISSUE THREAD

**Issue by tammojan** [lofar-astron/DP3#170]
As reported by @revoltek:
```
#################
/home/fdg/opt/lofar_190705/DPPP/src/Common/BaselineSelect.cc: In static member function ‘static casacore::Matrix<bool> DP3::BaselineSelect::convert(casacore::Table&, casacore::TableExprNode&, casacore::TableExprNode&, const string&, std::ostream&)’:
/home/fdg/opt/lofar_190705/DPPP/src/Common/BaselineSelect.cc:121:59: error: cannot convert ‘casacore::CountedPtr<casacore::MSSelectionErrorHandler>’ to ‘casacore::MSSelectionErrorHandler*’ in initialization
    MSSelectionErrorHandler* curHandler = MSAntennaParse::thisMSAErrorHandler;
                                                          ^
make[2]: *** [CMakeFiles/Common.dir/Common/BaselineSelect.cc.o] Error 1
make[1]: *** [CMakeFiles/Common.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
/home/fdg/opt/lofar_190705/DPPP/src/DPPP/MSWriter.cc: In member function ‘void DP3::DPPP::MSWriter::createMS(const string&, const DP3::DPPP::DPInfo&, unsigned int, unsigned int)’:
/home/fdg/opt/lofar_190705/DPPP/src/DPPP/MSWriter.cc:286:35: error: ‘SimpleOrderedMap’ was not declared in this scope
      newdesc.adjustHypercolumns (SimpleOrderedMap<String,String>(String()));
                                  ^
/home/fdg/opt/lofar_190705/DPPP/src/DPPP/MSWriter.cc:286:58: error: expected primary-expression before ‘,’ token
      newdesc.adjustHypercolumns (SimpleOrderedMap<String,String>(String()));
                                                         ^
/home/fdg/opt/lofar_190705/DPPP/src/DPPP/MSWriter.cc:286:65: error: expected primary-expression before ‘>’ token
      newdesc.adjustHypercolumns (SimpleOrderedMap<String,String>(String()));
                                                                ^
make[2]: *** [CMakeFiles/DPPP_OBJ.dir/DPPP/MSWriter.cc.o] Error 1
make[1]: *** [CMakeFiles/DPPP_OBJ.dir/all] Error 2
make: *** [all] Error 2
###################
```

**Comment by tammojan** [comment]
This is caused by casacore commit casacore/casacore@a157505, which is in the casacore master but not released yet. Probably DPPP needs to move from SimpleOrderedMap to the STL counterpart.

**Comment by tammojan** [comment]
In fact the offending casacore change was released in 3.1.1. I have a PR almost ready.

**Comment by tammojan** [comment]
Unfortunately this will require an update of casacore (though you can get a quick fix by manually updating the version in `casacore/casa/version.h`).
